accord-core/src/test/java/accord/verify/StrictSerializabilityVerifier.java

```diff
@@ -606,7 +623,9 @@ void insert(Step step)
                 steps = Arrays.copyOf(steps, Math.max(i + 1, steps.length * 2));
 
             steps[i] = step;
-            while (--i >= 0 && steps[i] == null) {}
+            while (--i >= 0 && steps[i] == null)
```

Comment:
unnecessary reformatting

Reply:
Seems that this recently changed https://cassandra.apache.org/_/development/code_style.html

> { and } are placed on a new line except when empty or opening a multi-line lambda expression.

I know there was a new vote/discussion on style changes and didn't keep up... will need to review...

for now ill revert